CONTRIBUTION_GUIDE.md

```diff
@@ -0,0 +1,124 @@
+
+## Fixing issues
+
+1. [Find an issue](https://github.com/aragon/aragon-apps/issues) that you are interested in.
+    - You may want to ask on the GitHub issue or in our [developer channel](https://discord.com/channels/672466989217873929/1036683514525003796) if anyone has already started working on the issue.
+2. Fork and clone a local copy of the repository.
+3. Make the appropriate changes for the issue you are trying to address or the feature that you want to add.
+4. Push the changes to the remote repository.
+5. Submit a pull request in Github, explaining any changes and further questions you may have.
+
+    We kindly ask that every PR follows this format and checks all checkboxes.
+
+    ```markdown
+    ## Description
+
+    ## Type of change
+
+    ## Checklist
+    [] I have selected the correct base branch.
+    [] I have performed a self-review of my own code.
+    [] I have commented my code, particularly in hard-to-understand areas.
+    [] I have made corresponding changes to the documentation.
+    [] My changes generate no new warnings.
+    [] Any dependent changes have been merged and published in downstream modules.
+    [] I ran all tests with success and extended them if necessary.
+    [] I have updated the CHANGELOG.md file in the root folder of the package after the [UPCOMING] title and before the latest version.
+    [] I have tested my code on the test network.
+    ```
+
+6. Wait for the pull request to be reviewer by the team.
+7. Make changes to the pull request if the maintainer recommends them.
+8. Celebrate your success after your pull request is merged! You will be able to claim your POAP for the hard work and join our Developer Inner Circle community.
+
+**Disclaimer**
+
+It's OK if your pull request is not perfect (no pull request is). The reviewer will be able to help you fix any problems and improve it.
+
+### Tips and Tricks
+
+Windows users may need to install the [windows-build-tools](https://www.npmjs.com/package/windows-build-tools) before installing this project's dependencies.
+
+### Documentation
+
+The documentation on how to run the front-end locally will be found within the `docs` folder of every repository.
+
+You can also access our [Developer Portal](https://devs.aragon.org) where you’ll learn about our architecture, how it works, and some tutorials which may be useful.
+
+### Style guide and development processes
+
+You can find our documentation [Style Guide here](https://www.notion.so/Documentation-Style-Guide-07c88cec18114b0aac88e8f0ba289976).
+
+For the frontends, we use [prettier](https://prettier.io/) and [eslint](https://eslint.org/) to automatically lint and format the project.
+
+For the contracts, we use [eth-lint](https://github.com/duaraghav8/Ethlint) to automatically lint the project.
```

Comment:
Aren't we using prettier as well?

For the SDK we are using Deno's builtin formatter